index.html

```diff
@@ -35,79 +49,197 @@
     <!-- main content on my web -->
 
     <main class="main">
-        <section class="section-1">
-            <h1 class="h-1">Hi there.<br>I'm Nico.</h1>
-            <h2 class="h-2">I'm a Sofware Developer</h2>
-            <p class="para-1">I can help you build a product,
-                feature or website Look through some of my work and experience!
-                If you like what you see and have a project you need coded,
-                don’t hesitate to contact me.</p>
-
-            <!-- social medias into <ul> -->
+        <div class="section-1-1">
+            <section class="section-1">
+                <h1 class="h-1" class="column-2">Hi there.<br>I'm Nico.</h1>
+                <h2 class="h-2" class="column-2">I'm a Sofware Developer</h2>
+                <p class="para-1" class="column-2">I can help you build a product,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded,
+                    don’t hesitate to contact me.</p>
+
+                <!-- social medias into <ul> -->
                 
-            <ul class="social-media">
-                <li><a href="" class="sm githubcat"><img src="GitHubcat.png" alt="My itHubcat"></a></li>
-                <li><a href="" class="sm linkedin"><img src="Linkedin.png" alt="My Linkedin"></a></li>
-                <li><a href="" class="sm angellist"><img src="Angellist.png" alt="My Angellist"></a></li>
-                <li><a href="" class="sm twitter"><img src="Twitter.png" alt="My Twitter"></a></li>
-                <li><a href="" class="sm medium"><img src="Medium.png" alt="My Medium"></a></li>
-            </ul>
+                <ul class="social-media">
+                    <li><a href="" class="sm githubcat"><img src="images/GitHubcat.png" alt="My itHubcat"></a></li>
+                    <li><a href="" class="sm linkedin"><img src="images/Linkedin.png" alt="My Linkedin"></a></li>
+                    <li><a href="" class="sm angellist"><img src="images/Angellist.png" alt="My Angellist"></a></li>
+                    <li><a href="" class="sm twitter"><img src="images/Twitter.png" alt="My Twitter"></a></li>
+                    <li><a href="" class="sm medium"><img src="images/Medium.png" alt="My Medium"></a></li>
+                </ul>
+            </section>
+                  <!-- background img -->
+        <section  class="div-img-3">
+            <img src="images/background-desktop-1.png" alt="" class="background-desktop-1">
```

Comment:
- [ ] [OPTIONAL] I highly suggest you remove the image and set it as a `background-image` property with CSS for the reasons discussed earlier. 🙂️